meta/documents/changelog_de.md

```diff
@@ -1,9 +1,12 @@
 # Release Notes für Nachnahme
 
-## X.X.X (2019-12-17)
+## 1.1.1 (2019-04-20)
 ### Geändert
 - Funktionalitäten hinzugefügt für Backend-Sichtbarkeiten und Backend-Name
 
+### Behoben
+- Ein Fehler der beim Logout aus dem MyAccount Bereich in Ceres 5 aufgetretten wurde behoben 
```

Comment:
"Ein Fehler der beim Logout aus dem MyAccount Bereich in Ceres 5 aufgetreten ist, wurde behoben"
@jawendel ;)